library/network/src/modules/NetworkPopup.rb

```diff
@@ -195,7 +195,7 @@ def NFSServer(selected)
         UI.OpenDialog(Label(_("Scanning for hosts on this LAN...")))
         # #71064
         # this works also if ICMP broadcasts are ignored
-        cmd = "/usr/sbin/rpcinfo -b mountd 1 | /usr/bin/cut -d ' ' -f 2 | /usr/bin/sort -u"
+        cmd = "/sbin/rpcinfo -b mountd 1 | /usr/bin/cut -f 2 | /usr/bin/sort -u"
```

Comment:
commit message does not explain why also -d is removed.

Reply:
You're right. Explained now.

> The delimiter in its output is a tab, not a space

It's because the two rpcinfo paths are actually different implementations with a slightly different output. See also https://github.com/yast/yast-nfs-client/pull/80 for full history.